cupertino-native/src/iosMain/kotlin/io/github/alexzhirkevich/cupertino/CupertinoContextMenu.kt

```diff
@@ -1,100 +0,0 @@
-///*
```

Comment:
If the file is commented out it doesn't mean that it should be deleted

Reply:
Why? You're using version control, if you need the commented out code, then find it in the repo history.

This library is riddled with commented-out and unused code, making it harder to debug and find things.

Checking in commented out code is a nasty code smell that fails to benefit the library.